octue/twined
# REF: Refactor tests

## Summary
* Simplify `BaseTestCase` path attribute
* Use `os.path.join` in tests
* Remove simple single-use variables from tests

In `tests.test_children`:
* Allow python object input in `Twine` instantiation, allowing test cases to be brought directly in to tests
* Bring test cases directly in to tests, allowing the entire test to be seen in one go in one test method (rather than having to look at two files to see what is being tested), and avoiding the need for parsing from JSON to python object in the test suite
* Shorten strings in test cases so they span fewer lines
* I've done this only to this module as I thought there might be a reason that I've missed to keep the test cases as JSON files or to not allow python objects to be used as the source in `Twine` instantiation (I didn't want to change further test files without checking first!)
